en_US/advanced/rebalancing.md

```diff
@@ -0,0 +1,364 @@
+Session goal: 0
+Session recipient  nodes: ['emqx2@127.0.0.1']
+Channel statistics:
+  current_connected: 0
+  current_sessions: 247
+  initial_connected: 233
+  initial_sessions: 247
+--------------------------------------------------------------------
+Node 'emqx2@127.0.0.1': rebalance coordinator
+Rebalance state: wait_health_check
+Coordinator node: 'emqx2@127.0.0.1'
+Donor nodes: ['emqx3@127.0.0.1']
+Recipient nodes: ['emqx2@127.0.0.1']
+Connection eviction rate: 5 connections/second
+Session eviction rate: 5 sessions/second
+```
+
+## API
+
+All the operations available from the CLI are also available from the API. Start/stop commands require a node
+as a parameter.
+
+### Enable Evacuation
+
+```
+curl -v -u admin:public -H "Content-Type: application/json" -X POST 'http://127.0.0.1:8081/api/v4/load_rebalance/emqx1@127.0.0.1/evacuation/start' -d '{"conn_evict_rate": 5, "sess_evict_rate": 5, "migrate_to": ["emqx3@127.0.0.1", "emqx2@127.0.0.1"]}'
+
+{"data":[],"code":0}
+```
+
+Possible body fields are `nodes`, `redirect_to`, `conn_evict_rate`, `migrate_to`, `wait_takeover`, and `sess_evict_rate`
+with the same meaning as in the corresponding [CLI command](#evacuation).
+
+### Disable Evacuation
+
+```
+curl -v -u admin:public -H "Content-Type: application/json" -X POST 'http://127.0.0.1:8081/api/v4/load_rebalance/emqx1@127.0.0.1/evacuation/stop'
+
+{"data":[],"code":0}
+```
+
+### Enable Rebalance
+
+```
+curl -v -u admin:public -H "Content-Type: application/json" -X POST 'http://127.0.0.1:8081/api/v4/load_rebalance/emqx1@127.0.0.1/start' -d '{"conn_evict_rate": 5, "sess_evict_rate": 5, "nodes": ["emqx1@127.0.0.1", "emqx2@127.0.0.1"]}'
+
+{"data":[],"code":0}
+```
+
+Possible body fields are `nodes`, `conn_evict_rate`, `sess_evict_rate`, `wait_takeover`, `wait_health_check`, `abs_conn_threshold`, `rel_conn_threshold`, `abs_sess_threshold`, and `rel_sess_threshold`
+with the same meaning as in the corresponding [CLI command](#rebalance).
+
+### Disable Rebalance
+
+```
+curl -v -u admin:public -H "Content-Type: application/json" -X POST 'http://127.0.0.1:8081/api/v4/load_rebalance/emqx1@127.0.0.1/stop'
+
+{"data":[],"code":0}
+```
+
+### Get Node-local Status
+
+```
+curl -s -u admin:public -H "Content-Type: application/json" -X GET 'http://127.0.0.1:8081/api/v4/load_rebalance/status'
+
+{
+  "status": "enabled",
+  "stats": {
+    "initial_sessions": 0,
+    "initial_connected": 0,
+    "current_sessions": 0,
+    "current_connected": 0
+  },
+  "state": "waiting_takeover",
+  "session_recipients": [
+    "emqx3@127.0.0.1",
+    "emqx2@127.0.0.1"
+  ],
+  "session_goal": 0,
+  "session_eviction_rate": 5,
+  "process": "evacuation",
+  "connection_goal": 0,
+  "connection_eviction_rate": 5
+}
+```
+
+### Get Cluster-wide Status
+
+```
+curl -s -u admin:public -H "Content-Type: application/json" -X GET 'http://127.0.0.1:8081/api/v4/load_rebalance/global_status'
+{
+  "rebalances": {},
+  "evacuations": {
+    "emqx1@127.0.0.1": {
```

Comment:
I have a question, this key appears to be mutable, which is unfriendly to json parsing, maybe we can like this:
```
{
  "evacuations": [
    {
      "node": "emqx1@127.0.0.1"
    }
  ]
}
```
@savonarola @zmstone what do you think

Reply:
@savonarola @zmstone ping again

Reply:
I'll make a PR

Reply:
https://github.com/emqx/emqx-enterprise/pull/1479